README.md

```diff
@@ -53,28 +54,63 @@ BLOG_USERNAME=abirch ./shell/docker-dev-watch.sh
 Visit the blog at:  
 http://localhost:4000
 
-### Linux:
-
-1. sudo apt-get install ruby2.3 ruby2.3-dev build-essential dh-autoreconf
-2. sudo gem update
-3. sudo gem install jekyll bundler
-4. Run 'jekyll -v' to check whether Jekyll is working
-5. bundle config path vendor/bundle
-6. gem install bundler
-7. sudo apt-get install libxslt-dev libxml2-dev zlib1g-dev
-8. sudo gem install nokogiri
-9. bundle install
-10. bundle exec jekyll serve
-11. Uncomment the lines in \_config.yml
-12. Access on http://localhost:4000
-
-To minify JS, run:
+### Native environment
+
+#### Prerequisites
+
+First, install Ruby and (if on Linux) a few build dependencies.
+
+On Linux:
+
+```bash
+sudo apt-get install ruby2.3 ruby2.3-dev build-essential dh-autoreconf libxslt-dev libxml2-dev zlib1g-dev
 ```
-npm run scripts
+
+On Windows, if you use Chocolatey, simply run `choco install ruby` in a PowerShell instance
+with elevated priveleges. If you don't use Chocolatey, you can see the Ruby website for
+[alternative ways to install Ruby](https://www.ruby-lang.org/en/documentation/installation).
+You don't need to install any other dependencies on Windows at this stage.
+
+Secondly, update Gem and install the Jekyll, Bundler and Nokogiri gems.
+
+On Linux:
+
+```bash
+sudo gem update
+sudo gem install jekyll bundler nokogiri
 ```
 
-To minify SCSS, run:
+On Windows, in a PowerShell instance with elevated priveleges:
+
+```bash
+gem update
+gem install jekyll bundler nokogiri
 ```
-npm run style
+
+Thirdly, configure Bundler to store project dependencies in `vendor/bundle`, and
+install the project dependencies.
+
+```bash
+bundle config path vendor/bundle
+bundle install
+```
+
+Finally, run `jekyll -v` to check whether Jekyll is working. If so, you're good to run the blog!
+
+#### Running in the native environment
+
+Once you've got all the prerequisites for your operating system, you can run the blog.
```

Comment:
OPTIONAL: You can add a link here to the appropriate section, e.g. `[installed the prerequisites](#prerequisites)`.

This lets you move the instructions somewhere else.

Why would you want to do that? 

Although running the application is the next logical step after installing it, my experience tends to be that I'll use the readme to remind myself of the run instructions far more often than I'll read the readme to install the application.
Accordingly having the most useful read-often bit at the top of the file with a link to read-once part is something I like to do.